lib/Newsletter/Scheduler/Welcome.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace MailPoet\Newsletter\Scheduler;
+
+use MailPoet\Models\Newsletter;
+use MailPoet\Models\SendingQueue;
+use MailPoet\Tasks\Sending as SendingTask;
+
+class Welcome {
```

Comment:
I think we could follow the naming pattern we started using with entities.
`MailPoet\Entities\NewsletterEntity`
So this would be:
`MailPoet\Newsletter\Scheduler\WelcomeScheduler` or maybe `MailPoet\Newsletter\Scheduler\WelcomeEmailScheduler`